.github/workflows/native-build-development.yml

```diff
@@ -43,21 +43,40 @@ jobs:
       - name: Check RSS
         env:
           QUICKSTART_VERSION: "1.0-SNAPSHOT"
-          RSS_THRESHOLD: "25000"
+          RSS_THRESHOLD: "40000"
           START_TIME_THRESHOLD: "1000"
+          BUCKET_NAME: ${{ secrets.BUCKET_NAME }}
+          REMOTE_OBJECT_NAME: "pmap.out"
+          API_KEY: ${{ secrets.API_KEY }}
```

Comment:
We may very soon have multiple API_KEY, can you prefix it with something like S3_API_KEY?

Reply:
@gsmet requested changes to the secret names. I proposed;

BUCKET_NAME -> IBM_COS_BUCKET_NAME
API_KEY -> IBM_COS_SERVICE_CREDENTIAL_API_KEY
SERVICE_INSTANCE_ID -> IBM_COS_SERVICE_CREDENTIAL_RESOURCE_INSTANCE_ID